doc/topics/pillar/index.rst

```diff
@@ -236,6 +236,51 @@ The resulting pillar will be as follows:
     For example, in the above scenario conflicting key values in ``services``
     will overwrite those in ``packages`` because it's at the bottom of the list.
 
+
+Referencing Other Pillars
+=========================
+
+.. versionadded:: ?????
+
+It is possible to reference pillar values, that are defined in other
+pillar files. To do that, the pillar with referenced pillars should
```

Comment:
This could be worded better. Perhaps:

> To do this, make sure that any pillar variables which are being referenced by other pillar SLS files are defined higher up in the pillar Top file, so that they are evaluated first.